GameServerLib/GameObjects/Spells/Spell.cs

```diff
@@ -30,7 +31,7 @@ public class Spell : ISpell
         public float CurrentCooldown { get; protected set; }
         public float CurrentCastTime { get; protected set; }
         public float CurrentChannelDuration { get; protected set; }
-        public uint FutureProjNetId { get; protected set; }
+        public List<IProjectile> Projectiles { get; protected set; }
```

Comment:
Consider different implementation like hash table.

Reply:
So, a dictionary?

Reply:
Should I make the key an int for their position in the dictionary or uint for the projectile's netID?

Reply:
Yeah, by NetId.